README.md

```diff
@@ -0,0 +1,30 @@
+# Ahoy::Intercom
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'ahoy-intercom'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install ahoy-intercom
+
+## Usage
+
+Set your environment variables `ENV['INTERCOM_APP_ID']` and `ENV['INTERCOM_ACCESS_TOKEN']`.
```

Comment:
"Set environment variables `INTERCOM_APP_ID` and `INTERCOM_ACCESS_TOKEN`."

And also add a link to Intercom doc on obtaining values for these variables